infrastructure/control-panel/.env.example

```
@@ -7,8 +7,16 @@ LOKI_PASSWORD=admin
 PUBLIC_REGISTRY_URL=https://registry.staging.metastate.foundation
 PUBLIC_CONTROL_PANEL_URL=http://localhost:5173
 
+# Provisioner
+PUBLIC_PROVISIONER_URL=http://localhost:3001
+
 # Notification Trigger (for Notifications tab proxy)
 NOTIFICATION_TRIGGER_URL=http://localhost:3998
+NOTIFICATION_TRIGGER_PORT=3998
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NOTIFICATION_TRIGGER_PORT key should go before the NOTIFICATION_TRIGGER_URL key

(UnorderedKey)